src/openstudio_lib/GridItem.cpp

```diff
@@ -1292,9 +1295,9 @@ void SystemCenterItem::paint(QPainter* painter, const QStyleOptionGraphicsItem*
     painter->drawRect(0, yOrigin, m_hLength * 100, 100);
   }
 
-  QFont f;
-  f.setPixelSize(20);
-  painter->setFont(f);
+  QFont font = painter->font();
+  font.setPointSize(14);
+  painter->setFont(font);
```

Comment:
I find that too small on my own machine.

Reply:
Yeah, I think these are going to be really hard to get perfect because everyone has different opinions + different screen sizes.  I agree that slightly larger text is a good idea for the Supply/Demand Equipment labels.

Reply:
Here are the differences between 1.2.0 (Qt5) and 1.4.0 (Qt6) on my laptop.  I have my laptop on the recommend settings for my display, resolution 1920x1080 and app scaling 125%.  The app scaling changed a bunch between Qt5 and Qt6, the icons look all low resolution (which is why I automatically upscaled them using imagemagick, svgs would be better).

https://user-images.githubusercontent.com/1327850/205473277-04126457-9b3c-485d-bf3d-15c34dda2b2a.mp4

Reply:
Here is the text at 20 pt on my computer, it gets cropped off.  I could try to find what is cropping it and extend instead.

![image](https://user-images.githubusercontent.com/1327850/205476125-22d314e6-34ba-439c-923c-1414a1641c4a.png)